sdk/storage/azure-storage-blob/azure/storage/blob/_encryption.py

```diff
@@ -498,6 +498,7 @@ def _generate_encryption_data_dict(
     :rtype: Dict[str, Any]
     """
     # Encrypt the cek.
+    wrapped_cek = bytes()
```

Comment:
Was this complaining about it being `None` type? Scary because this is in encryption but after looking through this file, seems like this change should have 0 effect.

Reply:
`wrapped_cek` is not an `Optional` type, so `pylint` and `mypy` wasn't having it :) Instead, I used default initialization for this as this is a safer bet than leaving as `None`. Either way, `wrapped_cek` immediately gets assigned next.

Reply:
Just to be clear - I believe we only have those two encryption protocols (V1, V2), so hence why I default initialized this to `bytes` type. The call to `kek.wrap_key(...)` on lines 503 and 508 are both returning a `bytes` object, hence why I'm default initializing to `bytes()`.

Reply:
See my comment on Queue for this.